src/colors/colors.stories.js

```diff
@@ -5,46 +5,46 @@ import { htmlStory } from "../story-helpers";
-    "Background lowest contrast": "--omrs-color-bg-lowest-contrast",
-
-    "Ink high contrast": "--omrs-color-ink-high-contrast",
-    "Ink medium contrast": "--omrs-color-ink-medium-contrast",
-    "Ink low contrast": "--omrs-color-ink-low-contrast",
-    "Ink lowest contrast": "--omrs-color-ink-lowest-contrast",
-    "Ink white": "--omrs-color-ink-white",
-
-    "Interaction plus two": "--omrs-color-interaction-plus-two",
-    "Interaction plus one": "--omrs-color-interaction-plus-one",
-    Interaction: "--omrs-color-interaction",
-    "Interaction minus one": "--omrs-color-interaction-minus-one",
-    "interaction minus two": "--omrs-color-interaction-minus-two",
-
-    Success: "--omrs-color-success",
-    "Success two": "--omrs-color-success-two",
-    Warning: "--omrs-color-warning",
-    "Warning two": "--omrs-color-warning-two",
-    Danger: "--omrs-color-danger",
-    "Danger two": "--omrs-color-danger-two",
-
-    "Color brand orange": "--omrs-color-brand-orange",
-    "Color brand violet": "--omrs-color-brand-violet",
-    "Color brand gold": "--omrs-color-brand-gold",
-    "Color brand teal": "--omrs-color-brand-teal",
-    "Color brand black": "--omrs-color-brand-black"
-  };
+export const options = {
+  "Primary blue": "--omrs-color-primary-blue",
+  "Text black": "--omrs-color-text-black",
+  "Inactive grey": "--omrs-color-inactive-grey",
+  "Inactive grey two": "--omrs-color-inactive-grey-two",
+  "Background canvas": "--omrs-color-background-canvas",
+  "Critical alert red": "--omrs-color-critical-alert-red",
+
+  "Background high contrast": "--omrs-color-bg-high-contrast",
+  "Background medium contrast": "--omrs-color-bg-medium-contrast",
+  "Background low contrast": "--omrs-color-bg-low-contrast",
+  "Background lowest contrast": "--omrs-color-bg-lowest-contrast",
+
+  "Ink high contrast": "--omrs-color-ink-high-contrast",
+  "Ink medium contrast": "--omrs-color-ink-medium-contrast",
+  "Ink low contrast": "--omrs-color-ink-low-contrast",
+  "Ink lowest contrast": "--omrs-color-ink-lowest-contrast",
+  "Ink white": "--omrs-color-ink-white",
+
+  "Interaction plus two": "--omrs-color-interaction-plus-two",
+  "Interaction plus one": "--omrs-color-interaction-plus-one",
+  Interaction: "--omrs-color-interaction",
+  "Interaction minus one": "--omrs-color-interaction-minus-one",
+  "interaction minus two": "--omrs-color-interaction-minus-two",
+
+  Success: "--omrs-color-success",
+  "Success two": "--omrs-color-success-two",
+  Warning: "--omrs-color-warning",
+  "Warning two": "--omrs-color-warning-two",
+  Danger: "--omrs-color-danger",
+  "Danger two": "--omrs-color-danger-two",
+
+  "Color brand orange": "--omrs-color-brand-orange",
+  "Color brand violet": "--omrs-color-brand-violet",
+  "Color brand gold": "--omrs-color-brand-gold",
+  "Color brand teal": "--omrs-color-brand-teal",
+  "Color brand black": "--omrs-color-brand-black"
+};
 
+storiesOf("OpenMRS Styleguide", module).add("Colors", () => {
```

Comment:
why did this file change?

Reply:
Because I exported the color options so they could be used in other stories like the icon story

Reply:
👍